packages/analytics/amplify_analytics_pinpoint_dart/pubspec.yaml

```diff
@@ -0,0 +1,38 @@
+name: amplify_analytics_pinpoint_dart
+description: A Dart-only implementation of the Amplify Analytics plugin for Pinpoint.
+version: 0.1.0
+homepage: https://docs.amplify.aws/lib/q/platform/flutter/
+repository:
+issue_tracker: https://github.com/aws-amplify/amplify-flutter/issues
+
+environment:
+  sdk: ">=2.17.0 <3.0.0"
+
+dependencies:
+  amplify_core: ^1.0.0-next.0
```

Comment:
question: Should these deps be referred in similar manner as other plugins?

Reply:
I think we can disregard like Dillon said and commit to changing this in the version bump PR